jetty-jaas/src/main/java/org/eclipse/jetty/jaas/JAASLoginService.java

```diff
@@ -243,50 +242,30 @@ else if (callback instanceof RequestParameterCallback)
 
             return _identityService.newUserIdentity(subject,userPrincipal,getGroups(subject));
         }
-        catch (LoginException e)
-        {
-            LOG.warn(e);
-        }
-        catch (IOException e)
-        {
-            LOG.warn(e);
-        }
-        catch (UnsupportedCallbackException e)
-        {
-           LOG.warn(e);
-        }
-        catch (InstantiationException e)
-        {
-            LOG.warn(e);
-        }
-        catch (IllegalAccessException e)
-        {
-            LOG.warn(e);
-        }
-        catch (ClassNotFoundException e)
+        catch (LoginException | IOException | UnsupportedCallbackException | InstantiationException | IllegalAccessException | ClassNotFoundException e)
```

Comment:
can this just be `catch(Exception)`?  what exception do you want to throw and not log?